src/markdown-pages/nr1-cli/nr1-nerdpack.mdx

```diff
@@ -0,0 +1,142 @@
+
+***
+
+## `nr1 nerdpack:clone` 
+### Clone an existing Nerdpack
+Duplicates an existing Nerdpack onto your local computer. You can clone an open source Nerdpack from [our GitHub repository](https://opensource.newrelic.com/explore-projects/). After choosing a git repository, this command performs the following actions so that you can start using the Nerdpack:
+1. Clones the repository.
+2. Sets the repository as remote upstream.
+3. Installs all of its dependencies (using npm).
+4. Generates a new UUID using your profile, and commits it.
+
+### Usage
+`$ nr1 nerdpack:clone OPTION` 
+
+### Options
+| | |
+|---|---|
+|  `-r, --repo=REPO`  | Repository location (either an HTTPS or SSH path). **(Required)** |
+|  `-p, --path=PATH`  | Determines the directory to clone to (defaults to the repository name). |
+|  `-f, --force`      |   Replaces destination folder if it exists. |
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+*** 
+
+## `nr1 nerdpack:serve`
+### Serve your Nerdpack locally
+Launches a server with your Nerdpack locally on your [New Relic One platform](https://one.newrelic.com/?nerdpacks=local), where it can be tested live. 
+
+For more on working with apps locally, see our guide on how to [serve, publish, and deploy documentation](/build-apps/publish-deploy/).
+
+### Usage
+`$ nr1 nerdpack:serve` 
+
+### Options
+| | |
+|---|---|
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+*** 
+
+## `nr1 nerdpack:uuid`
+### Show or generate your Nerdpack's UUID
+Prints the UUID (Universal Unique ID) of your Nerdpack, by default. The UUID determines what data the Nerdpack can access and who can subscribe to the Nerdpack. To deploy a Nerdpack you didn't make, you'll have to assign it a new UUID by using the `-g` or `--generate` option.
+
+For more details, see the [Access to data page](https://developer.newrelic.com/build-tools/new-relic-one-applications/guide-to-authentication--data-access--and-permissions#Appaccesstodata) or [our workshop](https://github.com/newrelic/nr1-workshop/blob/master/lab-cli/INSTRUCTIONS.md) on GitHub.
+
+### Usage
+`$ nr1 nerdpack:uuid` 
+
+### Options
+| | |
+|---|---|
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `-f, --force`       | If present, it will override the existing UUID without asking. |
+|  `-g, --generate`     | Generates a new UUID if not available. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+***
+
+## `nr1 nerdpack:publish`
+### Publish your Nerdpack
+Publishes your Nerdpack to New Relic. After publishing, you can use `nr1 nerdpack:deploy` to deploy it to a channel and make it available to users. 
+
+Please note:
+* If no additional parameters are passed in, this command will automatically deploy the Nerdpack onto the DEV channel. 
```

Comment:
I don't think this is true. I actually ran a test and I get.  -- Please confirm this.

C02Y40KMJHD4:jvs-stage-app jvajda$ nr1 nerdpack:deploy
✖  Error: Missing required flag:
-c, --channel CHANNEL  choose the channel you want to deploy the version
See more help with --help
C02Y40KMJHD4:jvs-stage-app jvajda$ nr1 nerdpack:deploy

Reply:
To clarify, this is referring to the `nr1 nerdpack:publish` command. Based on the description I got from `nr1 help nerdpack:publish`, if you publish your nerdpack, it will also deploy it by default. Otherwise, you can run the `nr1 nerdpack:publish --skip-deploy` option to skip this step. 

Though, I do agree it's confusing to say "After publishing, you can use `nr1 nerdpack:deploy` to deploy it to a channel and make it available to users" in this step. I'll remove it.